src/main/java/uk/gov/companieshouse/acsp/manage/users/controller/AcspMembershipController.java

```diff
@@ -36,11 +44,70 @@ public ResponseEntity<AcspMembership> getAcspMembershipForAcspAndId( final Strin
         return new ResponseEntity<>( membership, HttpStatus.OK );
     }
 
+    private void throwBadRequestWhenActionIsNotPermittedByOAuth2User( final User requestingUser, final AcspMembersDao membershipIdAssociation, final UserRoleEnum userRole ){
```

Comment:
we might have to remove all this code in few months when user scopes and permissions will come in.